src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -1,52 +1,18 @@
-//== Colors
```

Comment:
When I was working on the dark theme, I really really wanted to hack apart this file, but I didn't because it is a bootstrap file with the exception of the variables at the top. Are we for sure breaking away from bootstrap? If not, won't these changes will make the adoption of a later version of bootstrap more difficult?